src/sentry/migrations/0734_sunset_appstore_connect_integration.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 5.0.6 on 2024-07-08 16:12
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0733_relocation_provenance"),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name="latestappconnectbuildscheck",
+            unique_together=None,
+        ),
+        migrations.RemoveField(
+            model_name="latestappconnectbuildscheck",
+            name="project",
+        ),
+        migrations.DeleteModel(
+            name="AppConnectBuild",
+        ),
+        migrations.DeleteModel(
+            name="LatestAppConnectBuildsCheck",
+        ),
```

Comment:
This migration isn't safe to deploy currently. Migrations are deployed many minutes before code is. Removing tables that django could be using will result in requests/tasks failing. Generally what we do is remove the table from django state first, and then in a follow up pull request remove the tables from schema.

Reply:
If I set `is_post_deployment` to True, would it make it safe?

Reply:
Actually, I updated the migration to use `migrations.SeparateDatabaseAndState`